admin_guide/install/deploy_router.adoc

```diff
@@ -91,10 +91,116 @@ $ oadm router region-west -o yaml --images=myrepo/somerouter:mytag --credentials
 
 [NOTE]
 ====
-You can link:../../admin_guide/high_availability.html[set up a highly-available
+You can link:../high_availability.html[set up a highly-available
 router] on your OpenShift cluster using IP failover.
 ====
 
+[[wildcard-certificates]]
+== Wildcard Certificates
+
+Based on the implementation, you may be able to use a default certificate for
+TLS termination on your router.
+Default certificates are useful for implementing a wildcard certificate for 
+a domain. For example, if you have many routes of the format _<subdomain>.example.com_, you may
+want to install a router with a wildcard certificate for _*.example.com_.
+
+[NOTE]
+====
+Browsers only consider wildcards valid for subdomains one
+level deep. So in this example, the certificate would be valid for
+_a.example.com_ but not for _a.b.example.com_.
+====
+
+To provide the default certificate to the router, you must specify it in the
+create command with the `--default-cert` option. The certificate should be a
+concatenated file of the key, certificate, and any CA certificates that are
+required by the browser. The certificate should be in a form acceptable by the
+underlying router implementation. In the case of HAProxy, it should be a PEM
+based certificate:
+
+----
+$ oadm router --credentials="$KUBECONFIG" --default-cert=/full/path/to/certificate.pem
+----
+
+For HAProxy, if a default certificate is provided, it loads the default certificate first. The
+certificate that is loaded first is presented to any route that matches the
+CN on the certificate and any route that is secure but does not match any
+configured certificates.
+
+For example, if the default certificate is for _\*.example.com_ and a secure
+route for _www.foo.com_ is created with no certificates, the route is still
+written and the router serves the _*.example.com_ certficiate. This may result
+in a browser warning for users because the CN on the certificate does not match
+the URL.
+
+If no default certificate is supplied, the HAProxy router defaults to a generic,
+expired certificate that is provided in the base image.
+
+[[special-notes]]
+*Special Notes About Secure Routes*
+
+Currently, password protected key files are not supported. HAProxy prompts you
+for a password upon starting and does not have a way to automate this process.
+To remove a passphrase from a keyfile, you can run:
+
+----
+# openssl rsa -in <passwordProtectedKey.key> -out <new.key>
+----
+
+When creating a secure route, you must include your certificate files as a
+single line of text. Replace the existing line breaks with:
+
+----
+\n
+----
+
+== Routers
```

Comment:
You think whole `== Routers` section should be in the admin topic? I was thinking this was more Arch-y, esp since by this point in the topic it's already move past identifying the default HAProxy router, and installing it.